README.md

```diff
@@ -1,56 +1,39 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Learing Center
 
+> This project is what i was able to learn in Module 1 of Microverse Program. This is the Capstone Project
 
 ## Built With
 
-- Major languages
-- Frameworks
+- HTML & CSS
+- Javascript
 - Technologies used
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+[Live Demo](https://ernestkorir.github.io/Capstone-1/)
 
-## Getting Started
+## Loom Video
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Loom Video](https://www.loom.com/share/7225df2afc564e04aaebc5995effe2ed)
```

Comment:
- [ ] Please remember to highlight a piece of the code the you find interesting or proud of making